docs/examples/event-exporter.md

```diff
@@ -0,0 +1,271 @@
+| image    | Docker image of the event exporter.                    | `apache/skywalking-kubernetes-event-exporter:latest` |.                                                                                                           | `"skywalking-system-oap.skywalking-system:11800"`                       |
+| replicas | Number of event exporter pods.                         | `1`                                                  |
+| config   | Configuration of filters and exporters in YAML format. | `""`                                                 |
+
+Please note: if you ignore the `config` field, no filters or exporter will be created.
+
+This is because the EventExporter controller creates a configMap for all `config` values and
+attach the configMap to the event exporter container as configuration file.
+Ignoring the `config` field means an **empty** configuration file (with value `"""`) is provided to the event exporter.
+
+## Status
+
+| name              | description                                                                |
+|-------------------|----------------------------------------------------------------------------|
+| availableReplicas | Total number of available event exporter pods.                             |
+| conditions        | Latest available observations of the underlying deployment's current state |
+| configMapName     | Name of the underlying configMap.                                          |
+
+## Configuration
+
+The event exporter supports reporting specific events by different exporters.
+We can add filter configs to choose which events we are interested in,
+and include exporter names in each filter config to tell event exporter how to export filtered events.
+
+An example configuration is listed below:
+
+```yaml
+filters:
+  - reason: ""
+    message: ""
+    minCount: 1
+    type: ""
+    action: ""
+    kind: "Pod|Service"
+    namespace: "^default$"
+    name: ""
+    service: "[^\\s]{1,}"
+    exporters:
+      - skywalking
+
+exporters:
+  skywalking:
+    template:
+      source:
+        service: "{{ .Service.Name }}"
+        serviceInstance: "{{ .Pod.Name }}"
+        endpoint: ""
+      message: "{{ .Event.Message }}"
+    address: "skywalking-system-oap.skywalking-system:11800" 
+```
+
+### Filter Config
+
+| name      | description                                                                                                                         | example          |
+|-----------|-------------------------------------------------------------------------------------------------------------------------------------|------------------|
+| reason    | Filter events of the specified reason, regular expression like `"Killing\|Killed"` is supported.                                    | `""`             |.                                                                                                           | `"skywalking-system-oap.skywalking-system:11800"`                       |
```

Comment:
```suggestion
| reason    | Filter events of the specified reason, regular expression like `"Killing\|Killed"` is supported.                                    | `""`             |
```

The extra field `"skywalking-system-oap.skywalking-system:11800"` will not be rendered by markdown, could you please check it?

Reply:
Thx, updated!

Reply:
Hi @jichengzhi, it looks like the field `"skywalking-system-oap.skywalking-system:11800"` still exists, could you please delete it?

Reply:
Sorry I missed that before, should be removed now.